src/screens/Categories/WishList.jsx

```diff
@@ -0,0 +1,94 @@
+import {Text, View, FlatList, ScrollView, SectionList, StyleSheet,} from "react-native";
+  import React, { Component } from "react";
+  import { containers } from "../../styles/WishlistScreen/Screen_Wishlist";
+  import { titles } from "../../styles/WishlistScreen/Screen_Wishlist";
+  import WishedCard from "../../components/WishedCard";
+  
+  const sections = [
+    {
+      title: "Wishlist",
+      data: [
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 3.5,
+          price: 99.99,
+          reviews: 125,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
+        },
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 5,
+          price: 49.99,
+          reviews: 205,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
+        },
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 3.5,
+          price: 99.99,
+          reviews: 125,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
+        },
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 3.5,
+          price: 99.99,
+          reviews: 125,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
+        },
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 3.5,
+          price: 99.99,
+          reviews: 125,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
+        },
+        {
+          name: "Iron Man",
+          edition: "#65f",
+          rating: 3.5,
+          price: 99.99,
+          reviews: 125,
+          image: "https://findgure.s3.amazonaws.com/products/AlexG_1.png",
```

Comment:
let's make a new file with all of your links and name it here please, try to name your links with a related name.

Reply:
We'll be working on this, because we changed how the data is used, now we call it from a server and we will no longer be using the info in that format.

Reply:
Got it!